app/src/ui/changes/sidebar.tsx

```diff
@@ -113,6 +113,10 @@ export class ChangesSidebar extends React.Component<IChangesSidebarProps, void>
     this.props.dispatcher.ignore(this.props.repository, pattern)
   }
 
+  private onRevealInFileManager = (path: string) => {
```

Comment:
Can we document this please, specifically with information about how the path parameter is expected to be a path relative to the repository root.